src/index.ts

```diff
@@ -72,12 +72,13 @@ export class SalesforceConnector extends CoreConnector {
     return this.setConnection(conn)
   }
 
-  public async handle(req: Request, res: Response, next: NextFunction) {
+  public async handle(req: Request, res: Response) {
     if (req.method === 'GET' && req.path === AUTHPATH) {
       await this.handleAuthCode(req.query.code, res)
+      return true
+    } else {
```

Comment:
if (req.method === 'GET' && req.path === AUTHPATH) {
      await this.handleAuthCode(req.query.code, res)
      return true
}
return false